### DISCUSSION TITLE
InventoryItemChangeEvent inventory is always null

### DISCUSSION THREAD

**Discussion by leNicDev** [Minestom/Minestom#1666] [Help] [answered]
When updating an item inside a player inventory, `InventoryItemChangeEvent#getInventory` always returns null. The same thing happens with `PlayerInventoryItemChangeEvent`.

Branch: `master`

**Comment by TogAr2** [comment, marked as answer]
`getInventory()` returns `null` when the inventory is the players own inventory, which you can get by using `player.getInventory()`.
